easybuild/framework/easyconfig/default.py

```diff
@@ -116,7 +116,7 @@
     'stop': [None, 'Keyword to halt the build process after a certain step.', BUILD],
     'tests': [[], ("List of test-scripts to run after install. A test script should return a "
                    "non-zero exit status to fail"), BUILD],
-    'unpack_options': [None, "Extra options for unpacking source", BUILD],
+    'unpack_options': ['', "Extra options for unpacking source", BUILD],
```

Comment:
wrt `[]`, you mean as opposed to hmm, i don't know `unwanted_env_vars` below?

wrt `None`, ok, makes sense